src/types/AppointmentFormData.tsx

```diff
@@ -1,14 +1,25 @@
-type AppointmentFormData = {
-  firstName: string
-  lastName: string
-  phoneNum: string
+export type AppointmentFormData = {
+  first_name: string
+  last_name: string
+  mobile_phone: string
   email: string
   dob: string
-  apptDate: string
-  apptTime: string
-  apptType: string
-  optionalDescription: string
-  emergency: string
+  requested_date: string
+  requested_time: string
+  appointment_type: string
+  description: string
+  is_emergency: string
 }
 
-export default AppointmentFormData
+export interface Appointment {
```

Comment:
- Think we should discuss about the usage of types and interfaces in our project. I believe that we should use interface only when we need any interface specific functionalities like Declaration Merging or extending. If the only purpose it to declare an object structure, types are sufficient. But, I understand at the same time that it is sometimes a personal choice. Regardless, we could discuss and agree upon the usage and possible reasons as we don't need both `AppointmentFormData` and `Appointment` as their structure is the same.

Leaving a few links for reference:
1. [TypeScript: Should I use Types or Interfaces?](https://levelup.gitconnected.com/typescript-should-i-use-types-or-interfaces-44dd44ae5180#:~:text=Both%20interfaces%20and%20types%20are,between%20Interfaces%20and%20Types%E2%9C%85.)
2. [TypeScript: Should you use Types or Interfaces?](https://www.youtube.com/watch?v=zM9UPcIyyhQ)
3. [Interfaces vs Type Aliases: what's the difference?](https://www.youtube.com/watch?v=QYO-sieqLD4)

- Could you please update the extension of the file to `ts`.